readme.md

```diff
@@ -82,6 +104,15 @@ var Panda = function(name, age) {
 
 Your Answer:
 ```text
+var eat_bamboo = function(){
+var this = self
+
+for (i=0; i<self.num_bamboo_eaten.length; i++){
+  self.num_bamboo_eaten + 1
+  }
+}
```

Comment:
This will double the amount of bamboo eaten, each time it is called.  If I had eaten 10 pieces of bamboo, running this method will have me eating another 10 pieces of bamboo.  We just asked you to add one.

`self` is used when we need to access context outside of the function.  Used in this way, inside the function, `self` _is_ `this`, and therefore is unnecessary.